css/card.css

```diff
@@ -0,0 +1,112 @@
+.card-container {
+    border-radius: 5px;
+    background-color: #ffffffed;
+    width: 23rem;
+    height: 24.3rem;
+    box-shadow: 0px 2px 5px 1px palevioletred;
```

Comment:
Please use hex code values for consistency.

Reply:
done !!